src/storage-manager.coffee

```diff
@@ -149,6 +149,9 @@ class _StorageManager
 
     if @expectations?
       @handleExpectations(loader)
+      loader.loadOptions.returnValues ?= {}
```

Comment:
You can deprecate the line below by making this

`loader.loadOptions.returnValues ?= abort: ->`